src/main/scala/org/scalajs/dom/raw/Idb.scala

```diff
@@ -123,7 +124,7 @@ class IDBObjectStore extends js.Object {
 }
 
 trait IDBVersionChangeEventInit extends EventInit {
-  var newVersion: js.UndefOr[Int] = js.undefined
+  var newVersion: js.UndefOr[Int | Null] = js.undefined
```

Comment:
The convention so far has been to use the boxed classes when we have primitive-or-null. So this should be
```suggestion
  var newVersion: js.UndefOr[Integer] = js.undefined
```
In a future where Scala has explicit nulls, that will change. But we're not there yet.

Reply:
Np. Updated.